dotnet/src/SemanticKernel/AI/OpenAI/Clients/OpenAIClientAbstract.cs

```diff
@@ -132,14 +141,15 @@ protected virtual void Dispose(bool disposing)
     // HTTP user agent sent to remote endpoints
     private const string HTTPUseragent = "Microsoft Semantic Kernel";
 
-    private async Task<T> ExecutePostRequestAsync<T>(string url, string requestBody)
+    private async Task<T> ExecutePostRequestAsync<T>(string url, string requestBody, CancellationToken cancellationToken = default)
     {
         string responseJson;
 
         try
         {
             using HttpContent content = new StringContent(requestBody, Encoding.UTF8, MediaTypeNames.Application.Json);
-            HttpResponseMessage response = await this.HTTPClient.PostAsync(url, content);
+
```

Comment:
Change spacing here to make rebase with Tao's changes easier. And then rebase after Tao's changes land.